bluzelle/assetlist.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "bluzelle",
+  "assets": [
+    {
+      "description": "The native token of Bluzelle",
+      "denom_units": [
+        {
+          "denom": "ubnt",
+          "exponent": 0
+        },
+        {
+          "denom": "bnt",
+          "exponent": 6
+        }
+      ],
+      "base": "ubnt",
+      "name": "Bluzelle",
+      "display": "blz",
+      "symbol": "BLZ",
+      "logo_URIs": {
+        "jpg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.jpg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.svg"
+      },
+      "coingecko_id": "bluzelle",
+      "keywords": [
+        "bluzelle",
+        "game"
+      ]
+    },
+    {
+      "description": "The ELT token of Bluzelle's Gamma 4 Gaming Ecosystem",
+      "denom_units": [
+        {
+          "denom": "uelt",
+          "exponent": 0
+        },
+        {
+          "denom": "elt",
+          "exponent": 6
+        }
+      ],
+      "base": "uelt",
+      "name": "ELT",
+      "display": "elt",
+      "symbol": "ELT",
+      "logo_URIs": {
+        "jpg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.jpg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.svg"
+      },
+      "coingecko_id": "",
+      "keywords": [
+        "elt"
+      ]
+    },
+    {
+      "description": "The game token of Bluzelle's Gamma 4 Gaming Ecosystem",
+      "denom_units": [
+        {
+          "denom": "ug4",
+          "exponent": 0
+        },
+        {
+          "denom": "g4",
+          "exponent": 6
+        }
+      ],
+      "base": "ug4",
+      "name": "G4",
+      "display": "g4",
+      "symbol": "G4",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/g4.png",
```

Comment:
Strongly preferred to have an exact square aspect ratio for the image to avoid morphing in various displays